### DISCUSSION TITLE
Feature Request for Nemo: Add a Single Window mode/setting/option

### DISCUSSION THREAD

**Discussion by rkdawenterprises** [linuxmint/discussions#351] [Ideas]
Coming from Xfce/Thunar to Cinnamon/Nemo. Thunar has a setting where opening a folder outside of an existing Nemo window (like clicking on desktop icon) will open it in a new tab, not open a whole new window.

**Comment by Jopp-gh** [comment]
You can mitigate this if you uncheck some setting in Nemo's own preferences. 

I can't tell you what option bc I don't use the english user interface. 

The option should be "behavior".
There you uncheck sth.  like "open every folder in a separate window"

Also there's another option to "automatically open a new folder for mounted devices".uncheck that one aswell

**Reply by rkdawenterprises** [reply]
Thank you for the suggestion, but I already have these options "unchecked". It does not mitigate the opening of multiple windows.

![Screenshot from 2024-03-09 16-00-22_annotated](https://github.com/linuxmint/discussions/assets/108582693/99213a6d-c4d3-4f8b-9c2d-be561bae3003)

**Reply by Jopp-gh** [reply]
Agreed, therefore I added my two cents to your request. Multiple windows are confusing and tabs definitely to prefer. 

Consider,  I used the wording "mitigate" and not  "final solution".

**Comment by anaximeno** [comment]
There's a way this can be achieved for now (in **LM 21.3**, **LMDE 6**, or greater):
1. On the default menu write `nemo`
2. Right-click on the nemo file manager application
3. A submenu should open, click on the 'properties' option
4. It will open a small window, where it is **Command:** add `nemo --existing-window %U` instead of the default text that is there
5. Then click accept to conclude the change

If you want to revert this follow all steps from 1 to 3 and put `nemo %U` back in the Command input.

**Comment by Dan-cer** [comment]
Great to know! I have written a request about it somewhere, and here is the solution.
On thing I want to ask additionally: While Nemo usually opens in the same position, klicking on i.e. download folder button within browser (while Nemo is minimized) brings Nemo to forground but on a different position. Can that be fixed too?
Update:
I found the answer by myself:
In "Windows" settings open the tab "Behavior" and in the second section set "Position of new opened window" to "**cursor**" or "manually".
